egui/src/widgets/plot/mod.rs

```diff
@@ -1094,3 +1212,36 @@ impl PreparedPlot {
         }
     }
 }
+
+/// Returns next bigger power in given base
+/// e.g.
+/// ```ignore
+/// next_power(0.01, 10) == 0.01
+/// next_power(0.02, 10) == 0.1
+/// next_power(0.2,  10) == 1
```

Comment:
Unfortunately, that would require making this function public -- and I'm not sure if that's worth it just for doc tests (and where it would belong in that case).

Reply:
Oh right, you cannot doctest private functions 🤦 

But perhaps this could belong in `emath` anyway